apps/envited-x-data-space/common/validator/shacl/shacl.constants.ts

```diff
@@ -1,19 +1,24 @@
 import { Schema } from './shacl.types'
 
 export const SCHEMA_MAP = {
-  [Schema.default]: '/shaclSchema.ttl',
+  [Schema.automotiveSimulator]: '/schemas/automotive-simulator_shacl.ttl',
   [Schema.environmentModel]: '/schemas/environment_shacl.ttl',
   [Schema.general]: '/schemas/general_shacl.ttl',
   [Schema.georeference]: '/schemas/georeference_shacl.ttl',
   [Schema.gx]: '/schemas/gx_shacl.ttl',
   [Schema.hdmap]: '/schemas/hdmap_shacl.ttl',
+  [Schema.leakageTest]: '/schemas/leaking-test_shacl.ttl',
   [Schema.manifest]: '/schemas/manifest_shacl.ttl',
+  [Schema.marketplaceInfo]: '/schemas/marketplace-info_shacl.ttl',
```

Comment:
@jdsika do we still have this marketplace-info_shacl.ttl? I don't see it in the ontology repository